features/barprop.yml

```
@@ -0,0 +1,7 @@
+name: BarProp
+description: "BarProp is implemented by browser UI such as the address bar or scrollbars, historically to represent whether the user interface component was visible. For privacy reasons, the spec was changed to make `visible` represent whether is a popup or not. Apple has not yet implemented the change."
```

Two things here:

- Following from my suggestion to widen the set of keys, we have _just_ enough room to just name everything rather than refer to the interface.
- Never mention support or standards status in the description. There's an obscure Greek god that regards this as a kind of hubris, who makes documentation go out of date just after you stop thinking about it.

```suggestion
description: The `visible` property of the `window.locationbar`, `window.menubar`, `window.personalbar`, `window.scrollbars`, `window.statusbar`, and `window.toolbar` APIs historically represented the visibility of a browser user interface. For privacy reasons, it now represents whether the window is a pop-up.
```